### PR TITLE
Sport shoes store pull request

### PR DESCRIPTION
**Pull request by Lkupybida**
added code and gitignore

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,154 @@
+#include <iostream>
+#include "string.h"
```
Comment (bogdan9876) [inline main.cpp:2]: space before using namespace std

File: main.cpp [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+using namespace std;
+
+class Sneakers
+{
+private:
```
Comment (bogdan9876) [inline main.cpp:14]: the first public, the second protected, the third private

File: main.cpp [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    cout << "Size: " << size << endl;
+    cout << "Price: " << price << endl;
+    cout << "Quantity: " << quantity << endl;
+    cout << "Number of sales: " << numberOfSales << endl;
+}
+
+class SportsShoesShop
+{
+    friend class Sneakers;
+private:
```
Comment (bogdan9876) [inline main.cpp:68]: the first public, the second protected, the third private

**Review by bogdan9876 — CHANGES_REQUESTED** [review]
між деякими методами є пробіл , а між іншими немає

**Review by bogdan9876 — APPROVED** [review]

**Review by bogdan9876 — APPROVED** [review]

File: main.cpp [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    Sneakers array[2];
```
Comment (zveres-ss) [inline main.cpp:69]: 2 is magic variable, make it as constant

**Pull request merged**